src/network/networkd-netdev-gperf.gperf

```diff
@@ -53,6 +54,9 @@ Tunnel.CopyDSCP,             config_parse_bool,                  0,
 Tunnel.EncapsulationLimit,   config_parse_encap_limit,           0,                             offsetof(Tunnel, encap_limit)
 Peer.Name,                   config_parse_ifname,                0,                             offsetof(Veth, ifname_peer)
 Peer.MACAddress,             config_parse_hwaddr,                0,                             offsetof(Veth, mac_peer)
+Peer.PublicKey,              config_parse_wireguard_public_key,  0,                             0
+Peer.AllowedIPs,             config_parse_wireguard_ips,         0,                             0
```

Comment:
I'd probably name the "PublicKey" option "WiregardPublicKey" or so if it is in the generic [Peer] section. It's fine to leave fully generic options with generic names (i.e. AllowedIPs= is pretty generic I'd say), but I'd guess that the wireguard public keys would be in a different format, than let's say ipsec would have them, hence I'd prefix the setting here to indicate that this option is supposed to be in wireguard's format, and thing else.